maps/map_files/generic/Admin_level.dmm

```diff
@@ -10709,23 +9090,23 @@ aa
 wW
 KW
 FB
-Mt
-Mt
-iU
-iU
-iU
-iU
-ll
-ly
-lk
-ly
-mk
-iU
-iU
-iU
-iU
-Mt
-Mt
+Jl
```

Comment:
an issue of using /area/almayers on adminlevel is that the admin level has no lighting, and almayer area's don't have use dynamic lighting off. this results in the lack of lighting around the shuttles, which looks kinda goofy. probably need area/adminlevel subtypes for the areas shuttles land on admin z level